codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/MemberModel.java

```diff
@@ -385,6 +385,30 @@ public String getFluentSetterDocumentation() {
         return docBuilder.toString();
     }
 
+    public String getDefaultConsumerFluentSetterDocumentation() {
```

Comment:
Should we define this in Poet or here?

Reply:
Seems like all the other documentation definitions are here...

Reply:
I'm fine with this way, but I kinda like what Varun did in his pagination PR having a PaginationDocs.java file with the docs related to pagination